audit/var-45.md

```diff
@@ -1,4 +1,4 @@
-# Chain Variable Transaction 45
+# Chain Variable Transaction 44
```

Comment:
Why was this changed? We're on var nonce 45...

Reply:
Have no idea. Will remove this file